Uninitialized variable in unit tests

**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/HK/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Compiling the latest HK app with the latest GCC version (12.x) produces a warning about uninitialized variable in the unit test:

```
[ 93%] Building C object apps/hk/unit-test/CMakeFiles/coverage-hk-hk_app-testrunner.dir/hk_app_tests.c.o
/home/joe/code/cfecfs/github/apps/hk/unit-test/hk_app_tests.c: In function ‘Test_HK_HousekeepingCmd’:
/home/joe/code/cfecfs/github/apps/hk/unit-test/hk_app_tests.c:1235:5: error: ‘DummyMsg’ may be used uninitialized [-Werror=maybe-uninitialized]
 1235 |     HK_HousekeepingCmd(&DummyMsg);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/joe/code/cfecfs/github/apps/hk/unit-test/hk_app_tests.c:24:
/home/joe/code/cfecfs/github/apps/hk/unit-test/../fsw/src/hk_app.h:181:6: note: by argument 1 of type ‘const CFE_MSG_CommandHeader_t *’ {aka ‘const struct CFE_MSG_CommandHeader *’} to ‘HK_HousekeepingCmd’ declared here
  181 | void HK_HousekeepingCmd(const CFE_MSG_CommandHeader_t *Msg);
      |      ^~~~~~~~~~~~~~~~~~
```

**To Reproduce**
Add HK to latest CFS bundle, build using default config.  

**Expected behavior**
Should build cleanly

**System observed on:**
Debian (Latest version, with GCC 12.2)

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/unit-test/hk_app_tests.c
+++ b/unit-test/hk_app_tests.c
@@ -1231,6 +1231,8 @@ void Test_HK_HousekeepingCmd(void)
     HK_AppData.CombinedPacketsSent = 4;
     HK_AppData.MemPoolHandle       = HK_UT_MEMPOOL_1;
 
+    memset(&DummyMsg, 0, sizeof(DummyMsg));
+
     /* Act */
     HK_HousekeepingCmd(&DummyMsg);
 

